tests/test_x509.py

```diff
@@ -1458,17 +1456,14 @@ def test_build_cert_printable_string_country_name(self, backend):
 
         cert = builder.sign(issuer_private_key, hashes.SHA256(), backend)
 
-        parsed, _ = decoder.decode(
-            cert.public_bytes(serialization.Encoding.DER),
-            asn1Spec=rfc2459.Certificate()
-        )
-        tbs_cert = parsed.getComponentByName('tbsCertificate')
-        subject = tbs_cert.getComponentByName('subject')
-        issuer = tbs_cert.getComponentByName('issuer')
-        # \x13 is printable string. The first byte of the value of the
-        # node corresponds to the ASN.1 string type.
-        assert subject[0][0][0][1][0] == b"\x13"[0]
-        assert issuer[0][0][0][1][0] == b"\x13"[0]
+        parsed = Certificate.load(
+            cert.public_bytes(serialization.Encoding.DER))
+        tbs_cert = parsed['tbs_certificate']
+        subject = tbs_cert['subject']
+        issuer = tbs_cert['issuer']
+
+        assert subject.dump()[11] == b'\x13'[0]
+        assert issuer.dump()[11] == b'\x13'[0]
```

Comment:
Ah yes, it appears that `['value']` in this case is a `DirectoryString`, which is a `core.Choice` type. I think this would have to be revised to:

```python
assert parsed.subject.chosen[0][0]['value'].chosen.tag == 19
assert parsed.issuer.chosen[0][0]['value'].chosen.tag == 19
```

*I should note, I don't have this checked out, so be sure verify that any style suggestions I make here actually function properly, since I have not! :smile:*